### ISSUE TITLE
Running executable fails on Ubuntu 20.04

### ISSUE THREAD

**Issue by fhchl** [dsymbol/yt-dlp-gui#12]
I get

```
> ./Downloads/yt-dlp-gui/yt-dlp-gui
[16186] Error loading Python lib '/home/fhchl/Downloads/yt-dlp-gui/libpython3.10.so.1.0': dlopen: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.35' not found (required by /home/fhchl/Downloads/yt-dlp-gui/libpython3.10.so.1.0)
```

**Comment by dsymbol** [comment]
Should be fixed in latest [Release v14.05.2023](https://github.com/dsymbol/yt-dlp-gui/releases/tag/14.05.2023)
For more details https://github.com/orgs/pyinstaller/discussions/7541

**Comment by fhchl** [comment]
Can confirm that the fix works. Thanks!
